### PR TITLE
serentity.properties cleanup and improved readme instructions for installation

### PR DESCRIPTION
**Pull request by blangenberg**
- ensures line breaks are LF (unix-based) not CRLF (windows-based)
- ignores IDE metadata
- updates installation instructions based on my experience
- changes the `serenity.properties` settings for the chrome webdriver to reference a relative directory. referring to `/Users/vignesh/...` will not work for anyone else.

Unfortunately, the diff looks like I completely rewrote the files because they have CRLF line breaks currently :(

### CODE REVIEW

File: qa-ui-automation/README.md [hunk 1]
```diff
@@ -1,111 +1,101 @@
...
-
-
+# Web UI Automation Suite
+
+-> Project description goes here
+
+## Getting Started
+
+### Mac:
+1. Install Java: https://www.oracle.com/java/technologies/javase-downloads.html
```
Comment (blangenberg) [inline qa-ui-automation/README.md:8]: I wonder if we can use `brew` for the java install, too? I'm not installed that way.

File: qa-ui-automation/README.md [hunk 2]
```diff
@@ -1,111 +1,101 @@
-# Web UI Automation Suite
-
--> Project description goes here
-
-## Getting Started
-
-
-### Mac:
-1. Install java: http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
```
Comment (blangenberg) [inline qa-ui-automation/README.md:9]: Java 8 is really old. We should use the latest and greatest for new projects: version 15.

File: qa-ui-automation/README.md (hunk 1 above)
Comment (nvignesh1234) [reply qa-ui-automation/README.md:8]: yes brew can be used or we can download the same from https://www.oracle.com/java/technologies/javase-downloads.html

Comment (nvignesh1234) [reply qa-ui-automation/README.md:8]: @blangenberg there might be dependencies with junit and other related jar files.Lets me do a complete run with jdk 15 update in local and try to see it is safer to merge

File: qa-ui-automation/README.md (hunk 2 above)
Comment (nvignesh1234) [reply qa-ui-automation/README.md:9]: @blangenberg there might be dependencies with junit and other related jar files.Lets me do a complete run with jdk 15 update in local and try to see it is safer to merge

File: qa-ui-automation/README.md (hunk 1 above)
Comment (nvignesh1234) [reply qa-ui-automation/README.md:8]: brew is also good

**Review by nvignesh1234 — APPROVED** [review]

**Pull request merged**